src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.html

```diff
@@ -25,16 +28,19 @@ <h1 class="govuk-heading-l">Do you want a specific judge?</h1>
               <label class="govuk-label govuk-radios__label" for="specificJudge">No</label>
             </div>
             <div class="govuk-radios__conditional" [ngClass]="{'govuk-radios__conditional--hidden': specificJudgeSelection !=='No'}">
-              <div class="govuk-form-group">
+              <div class="govuk-form-group" [ngClass]="{'govuk-form-group--error': !!selectJudgeTypesError}">
                 <fieldset class="govuk-fieldset">
+                  <span class="govuk-error-message" *ngIf="selectJudgeTypesError">
+                    <span class="govuk-visually-hidden">Error:</span> {{selectJudgeTypesError}}
+                  </span>
                   <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
                     <h1 class="govuk-fieldset__heading">Select all judge types that apply</h1>
                   </legend>
-                  <div class="govuk-form-group" id="facilitiesList">
+                  <div class="govuk-form-group" id="judgeTypes">
                     <fieldset class="govuk-fieldset">
-                      <div class="govuk-checkboxes" data-module="govuk-checkboxes">
-                        <div class="govuk-checkboxes__item" *ngFor="let judgeType of hearingJudgeTypes">
-                          <input class="govuk-checkboxes__input" id="{{judgeType.key}}" name="judgecontract" type="checkbox" [value]="judgeType.value_en">
+                      <div class="govuk-checkboxes" formArrayName="judgeType" data-module="govuk-checkboxes">
+                        <div class="govuk-checkboxes__item" *ngFor="let judgeType of hearingJudgeTypes;index as pos" [formGroupName]="pos">
```

Comment:
[formGroupName]="pos"? so this index is used as formGroup? can this formGroupName be something else meaniful?

Reply:
Position is used because, in component, we generate dynamic FormArray with key. 
`judgeType: this.getJudgeTypeFormArray'  returns => [0:{selected: true,...},...]`